src/donor_dashboard/templates/donor_dashboard/list.html

```diff
@@ -4,6 +4,21 @@
 {% endblock title %}
 {% block content %}
     <div class="container">
+                
+        <!-- Notification -->
+        {% if messages %}
```

Comment:
Lines 8 to 20 can be replaced with ```{% include "partials/notifications.html" %}```

Reply:
Noticed this change and It was updated before merging.